enki/PhysicalEngine.h

```diff
@@ -372,6 +372,12 @@ namespace Enki
 		void collideWithStaticObject(const Vector &n, const Point &cp);
 		//! Dynamics for collision with that at point cp (on that) with a penetrated distance of dist,
 		void collideWithObject(PhysicalObject &that, Point cp, const Vector &dist);
+
+	public:
+		// ID is used when sharing a world over the network where it should be
```

Comment:
Please use Doxygen comments (//!), starting with one line short description, and optionally more lines of detailed descriptions.